.github/workflows/connector-ui-sanity-tests.yml

```diff
@@ -101,8 +101,8 @@ jobs:
       - name: Install latest compiler
         uses: actions-rs/toolchain@v1
         with:
-            toolchain: stable
-            override: true
+          toolchain: stable
+          override: true
```

Comment:
I don't think we need this to be specified, we use the same action in other workflows and haven't specified this input.

Reply:
I checked with  `connector-sanity-tests.yml` and yes, we don't need `override` to be set to `true`. But, `toolchain` version is recommended to be kept as it is.